### ISSUE TITLE
Upload an empty file returns incorrect message

### ISSUE THREAD

**Issue by yfdyh000** [mozilla/send#383]
Try upload an 0 bytes file to testing the functionality.

The Send page alert the `Uploading multiple files or a folder is currently not supported.` (`uploadPageMultipleFilesAlert`).

**Comment by pdehaan** [comment]
Not sure exactly what the code is doing, but I believe this is coming from [/frontend/src/upload.js:143](https://github.com/mozilla/send/blob/f1c894d14fce7a514428914ffa69b82355508f42/frontend/src/upload.js#L143), where we explicitly check for a file size of 0:

```js
event.originalEvent.dataTransfer.files[0].size === 0
```

**Comment by ericawright** [comment]
That check is in there in order to catch when the user uploads a folder. The "file" size comes through as 0 in that case as well.

**Comment by ehuggett** [comment]
I can upload 0byte files on production and dev.

I did manage to grab this screenshot from dev
![send_nan](https://user-images.githubusercontent.com/8090731/29754963-d874219a-8b87-11e7-98cb-3d5cc94fb0c3.png)

234B sounds plausible for posting an "empty" encrypted file (probably 32Bytes itself?) as an attachment on a form that also has a few metadata fields and headers (filename, content-type etc), its the total size of the form (and its file)?

NaN is of course "not a number", presumably something has been divided by the file size?

I tried to upload the 0byte file to github as "proof" that it really was empty, but github won't let me 😄 

![screenshot](https://user-images.githubusercontent.com/8090731/29755005-5ba4346a-8b88-11e7-90e9-f530de1c9996.png)
